tools/wpt/browser.py

```diff
@@ -364,8 +364,7 @@ def install_geckodriver_nightly(self, dest):
             os.unlink(package_path)
 
         return path
-
-    def version(self, binary=None, channel=None):
+    def version(self, binary=None, webdriver_binary=None, channel=None): # !!
```

Comment:
? # !!

Reply:
Yes, this line was irregular, `channel` is never passed and there's dead code. I left the comment to remember simplifying it first in a separate commit.